docs/src/snap/howto/cos-lite.md

```diff
@@ -0,0 +1,142 @@
+# Integrating with COS Lite
+
+**Canonical Kubernetes** includes the standard Kubernetes dashboard for
+monitoring your cluster. However, it is often advisable to have a monitoring
+solution which will run whether the cluster itself is running or not. It may
+also be useful to integrate monitoring into existing setups.
+
+To make monitoring your cluster a delightful experience, Canonical provides
+first-class integration between Canonical Kubernetes and COS Lite (Canonical
+Observability Stack). This guide will help you integrate a COS Lite
+deployment with a Canonical Kubernetes deployment.
+
+This document assumes you have a controller with an installation of
+Canonical Kubernetes. If this is not your case, refer to
```

Comment:
" If not your case..."
I think it's clearer to say something like
"If you have not yet installed Canonical Kubernetes, please see [the Install section of the documentation][] "